plugins/package/README.md

```diff
@@ -13,6 +13,20 @@ keytool -genkey -alias "signing key 2" -keystore signingkeys.pfx -storepass "key
 # To trust freetsa.org, download their CA cert and import into keystore
 keytool -import -alias "freetsa" -keystore signingkeys.pfx -storepass "keystore password" -file ~/Downloads/cacert.pem
 
+# Build a CPB
```

Comment:
Can we please move all this functionality to `corda-runtime-os` repo under: https://github.com/corda/corda-runtime-os/tree/release/os/5.0/tools/plugins ?

Reply:
Once this is merged, next task is to move the package plugin to `corda-runtime-os` repo.